src/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers, Redux } from 'react-redux';
```

Comment:
Please import `combineReducers` and `createStore` from `redux`. Otherwise, you will run into import errors in the future since `Redux` does not exist in `react-redux`
```
  import { combineReducers, createStore } from 'redux'
```

- You might wanna install `redux` as well since we are importing from it:
```
  npm install redux
```